README.md

```diff
@@ -8,7 +8,7 @@ Downloads](https://poser.pugx.org/willdurand/js-translation-bundle/downloads.png
 [![Latest Stable
 Version](https://poser.pugx.org/willdurand/js-translation-bundle/v/stable.png)](https://packagist.org/packages/willdurand/js-translation-bundle)
 
-A pretty nice way to expose your Symfony2 translation messages to your client
+A pretty nice way to expose your Symfony >4.0 translation messages to your client
```

Comment:
I suggest changing this to `Symfony`. this sentence does not need to duplicate the compatibility info. the composer.json is the source of truth for such thing.